### PR TITLE
make the repo Docusaurus compatible

### PR DESCRIPTION
**Pull request by Dr-Electron**
This PR makes the repo Docusaurus compatible.
Now we just need to decide where and how to add it to the wiki.

### CODE REVIEW

File: docs/sidebars.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+module.exports = {
+    docs: [
+        {
+            type: 'category',
+            label: 'Zebra IOTA Edge SDK',
+            collapsed: false,
+            items: [
+                'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-101-tutorial',
```
Comment (lucas-tortora) [inline docs/sidebars.js:19]:
```suggestion
                {
                    type: 'doc',
                    label: '101: Understanding Decentralized Identities',
                    id: 'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-101-tutorial'
                },
```

File: docs/sidebars.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial',
```
Comment (lucas-tortora) [inline docs/sidebars.js:20]:
```suggestion
                {
                    type: 'doc',
                    label: '102: Managing credentials (Holder Application)',
                    id: 'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial'
                },
```

File: docs/sidebars.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-103-tutorial',
```
Comment (lucas-tortora) [inline docs/sidebars.js:21]: {
                    type: 'doc',
                    label: '103: Verifying Credentials',
                    id: 'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-103-tutorial'
                },

File: docs/sidebars.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial',
```
Comment (lucas-tortora) [inline docs/sidebars.js:22]:
```suggestion
                {
                    type: 'doc',
                    label: '104: Device Onboarding',
                    id: 'tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial'
                },
```

File: docs/sidebars.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            ],
+        },
+        {
+            type: 'category',
+            label: 'Track And Trace Ledger API',
+            collapsed: false,
+            items: [
+                'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-101',
```
Comment (lucas-tortora) [inline docs/sidebars.js:30]:
```suggestion
                {
                    type: 'doc',
                    label: '101: Device Events',
                    id: 'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-101',
                },
```

File: docs/sidebars.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-102',
```
Comment (lucas-tortora) [inline docs/sidebars.js:31]:
```suggestion
                {
                    type: 'doc',
                    label: '102:  Device Events Recorded On the Tangle',
                    id: 'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-102',
                },
```

File: docs/sidebars.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+                'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-103',
```
Comment (lucas-tortora) [inline docs/sidebars.js:32]:
```suggestion
                {
                    type: 'doc',
                    label: '103: Digital Identity',
                    id: 'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-103',
                },
```

File: docs/sidebars.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+                'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-201',
```
Comment (lucas-tortora) [inline docs/sidebars.js:33]:
```suggestion
                {
                    type: 'doc',
                    label: '201: Business Events',
                    id: 'tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-201',
                },
```

File: docs/tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-201.md [hunk 9]
```diff
@@ -12,7 +12,7 @@ Business Events are complementary to Device Events, already introduced by the [1
 
 Typically, one or more Device Events (for instance RFID Read Device Events) yield to a Business Event. The transformation from a Device Event to a Business Event needs contextual or situational information that might depend on each application. Particularly, Business Events need *Reference ("Master") Data* to refer to. *Reference Data* can represent business locations, read points, devices, product classes, etc. In contrast to event visibility data, that is continuously generated, additions or changes to Reference Data happen sporadically.
 
-![Common Record of Truth](../images/tangle-supply-chain-common-record-of-truth.png "Common Record of Truth")
+![Common Record of Truth](/img/track-trace-ledger-api/tangle-supply-chain-common-record-of-truth.png "Common Record of Truth")
```
Comment (7opf) [inline docs/tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-201.md:15]: This has broken the markdown embed: https://github.com/Dr-Electron/Zebra-Iota-Edge-SDK/blob/docs/docusaurus/docs/tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-201.md

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-101-tutorial.md [hunk 10]
```diff
@@ -17,7 +17,7 @@ Based on its [technical definition](https://www.justaskgemalto.com/en/what-is-di
 The details collected about an identity (for example name, email address, DoB, credit card number, vaccinations taken for personal identities) during the identity creation phase represent what are called the **identity claims**. An identity is associated with zero, one or multiple claims.
 
 <p align="center">
-  <img src="images/image2.png" width="50%" alt="Decentralized Identity Overview">
+  <img src="/img/zebra-iota-edge-sdk/101/image2.png" width="50%" alt="Decentralized Identity Overview"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-101-tutorial.md:20]: Broken markdown embed https://github.com/Dr-Electron/Zebra-Iota-Edge-SDK/blob/docs/docusaurus/docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-101-tutorial.md

**Review by lucas-tortora — APPROVED** [review]
LGTME :) 
If we have the time, I'd like to create a  subsequent PR to edit the docs with our style guide, fix 2 links in zebra-iota-edge-sdk-101-tutorial.md and add some meta tags. 

I've added some comments so the sidebar doesn't look as busy, though these could be added in a separate PR as well.

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md [hunk 11]
```diff
@@ -61,12 +61,12 @@ Afterwards you need to go to the Android Studio and compile, install and execute
...
-In order to use the credential issuance part of this application (see [DeviceID tutorial](../104/zebra-iota-edge-sdk-104-tutorial.md#onboard-a-device)) you will need to set up a DataWedge profile bound to the `“org.iota.zebra.holder”` application as follows:
+In order to use the credential issuance part of this application (see [DeviceID tutorial](../zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md#onboard-a-device)) you will need to set up a DataWedge profile bound to the `“org.iota.zebra.holder”` application as follows:
 
 <p align="center">
-<img alt="Data Wedge Profile 1" src="images/image3.png" width="30%">
-<img alt="Data Wedge Profile 2" src="images/image10.png" width="30%">
-<img alt="Data Wedge Profile 3" src="images/image1.png" width="30%">
+<img alt="Data Wedge Profile 1" src="/img/zebra-iota-edge-sdk/102/image3.png" width="30%"/>
+<img alt="Data Wedge Profile 2" src="/img/zebra-iota-edge-sdk/102/image10.png" width="30%"/>
+<img alt="Data Wedge Profile 3" src="/img/zebra-iota-edge-sdk/102/image1.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md:69]: Broken img embeds in markdown: https://github.com/Dr-Electron/Zebra-Iota-Edge-SDK/blob/docs/docusaurus/docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md [hunk 12]
```diff
@@ -75,16 +75,16 @@ Note: On a non DataWedge device you can simulate a DataWedge scanning through th
...
 
 Under the first time experience wizard, once you enter a name associated with your new Identity, (last step of the wizard) the application will create a new DID, anchoring it to the IOTA Tangle. The private key that allows control of such an Identity is securely stored on the device.
 
 <p align="center">
-<img alt="Identity Creation 1" src="images/image9.png" width="30%">
-<img alt="Identity Creation 2" src="images/image6.png" width="30%">
-<img alt="Identity Creation 3" src="images/image26.png" width="30%">
+<img alt="Identity Creation 1" src="/img/zebra-iota-edge-sdk/102/image9.png" width="30%"/>
+<img alt="Identity Creation 2" src="/img/zebra-iota-edge-sdk/102/image6.png" width="30%"/>
+<img alt="Identity Creation 3" src="/img/zebra-iota-edge-sdk/102/image26.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md:87]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md [hunk 13]
```diff
@@ -161,9 +161,9 @@ Note: The user is both the subject and the holder of the identity and credential
 Note: At any point in time you can click on the button on top left of the main page of the application and restart the process of identity and credentials generation.
 
 <p align="center">
-<img alt="Credential Issuance 1" src="images/image24.png" width="30%">
-<img alt="Credential Issuance 2" src="images/image14.png" width="30%">
-<img alt="Credential Issuance 3" src="images/image27.png" width="30%">
+<img alt="Credential Issuance 1" src="/img/zebra-iota-edge-sdk/102/image24.png" width="30%"/>
+<img alt="Credential Issuance 2" src="/img/zebra-iota-edge-sdk/102/image14.png" width="30%"/>
+<img alt="Credential Issuance 3" src="/img/zebra-iota-edge-sdk/102/image27.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md:166]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md [hunk 14]
```diff
@@ -212,8 +212,8 @@ As a result the verifier can validate not only the credential itself, but also t
 Once a verifiable presentation has been generated it is encoded using a data matrix code. In our example we are using a 6 by 6 data matrix code that can later be scanned by the _Verifier Application_ (see below). If you double tap on the data matrix code you will be able to inspect the JSON-LD content that represents the credential presented.
 
 <p align="center">
-<img alt="Credential Presentation 1" src="images/image15.png" width="30%">
-<img alt="Credential Presentation 2" src="images/image5.png" width="30%">
+<img alt="Credential Presentation 1" src="/img/zebra-iota-edge-sdk/102/image15.png" width="30%"/>
+<img alt="Credential Presentation 2" src="/img/zebra-iota-edge-sdk/102/image5.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md:216]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-103-tutorial.md [hunk 15]
```diff
@@ -60,9 +60,9 @@ Afterwards you need to go to the Android Studio and compile, install and execute
 In order to run this application fully integrated with a Zebra Android scanning device you need to set up a DataWedge profile for the `“org.iota.zebra.verifier”` application as follows, or you can import it from [here](https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/blob/main/test/datawedge/datawedge.db).
 
 <p align="center">
-<img alt="Data Wedge Profile 1" src="images/image3.png" width="30%">
-<img alt="Data Wedge Profile 2" src="images/image7.png" width="30%">
-<img alt="Data Wedge Profile 3" src="images/image19.png" width="30%">
+<img alt="Data Wedge Profile 1" src="/img/zebra-iota-edge-sdk/103/image3.png" width="30%"/>
+<img alt="Data Wedge Profile 2" src="/img/zebra-iota-edge-sdk/103/image7.png" width="30%"/>
+<img alt="Data Wedge Profile 3" src="/img/zebra-iota-edge-sdk/103/image19.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-103-tutorial.md:65]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-103-tutorial.md [hunk 16]
```diff
@@ -82,9 +82,9 @@ You can see the steps from the application below. After scanning a Verifiable Pr
 Once the credential is deemed as valid it is shown and persisted on the inbox of the application (third capture below) for demonstration purposes. When implementing a concrete solution different actions could be taken when scanning a valid credential.
 
 <p align="center">
-<img alt="Credential Verification 1" src="images/image11.png" width="30%">
-<img alt="Credential Verification 2" src="images/image13.png" width="30%">
-<img alt="Credential Verification 3" src="images/image25.png" width="30%">
+<img alt="Credential Verification 1" src="/img/zebra-iota-edge-sdk/103/image11.png" width="30%"/>
+<img alt="Credential Verification 2" src="/img/zebra-iota-edge-sdk/103/image13.png" width="30%"/>
+<img alt="Credential Verification 3" src="/img/zebra-iota-edge-sdk/103/image25.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-103-tutorial.md:87]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md [hunk 17]
```diff
@@ -68,50 +68,50 @@ adb shell am startservice -a org.iota.zebra.device_id.intent.action.SCAN -e com.
 ```
 
 <p align="center">
-<img alt="Data Wedge Profile 1" src="images/image3.png" width="30%">
-<img alt="Data Wedge Profile 2" src="images/image18.png" width="30%">
-<img alt="Data Wedge Profile 3" src="images/image22.png" width="30%">
+<img alt="Data Wedge Profile 1" src="/img/zebra-iota-edge-sdk/104/image3.png" width="30%"/>
+<img alt="Data Wedge Profile 2" src="/img/zebra-iota-edge-sdk/104/image18.png" width="30%"/>
+<img alt="Data Wedge Profile 3" src="/img/zebra-iota-edge-sdk/104/image22.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md:73]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md [hunk 18]
```diff
@@ -68,50 +68,50 @@ adb shell am startservice -a org.iota.zebra.device_id.intent.action.SCAN -e com.
...
-The first step to onboard a device is to obtain a new DID document. This process is conducted through a wizard, and is similar to the one we already described for the [_Holder Application_](../102/zebra-iota-edge-sdk-102-tutorial.md#create-an-identity). See below.
+The first step to onboard a device is to obtain a new DID document. This process is conducted through a wizard, and is similar to the one we already described for the [_Holder Application_](../zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md#create-an-identity). See below.
 
 <p align="center">
-<img alt="Identity device 1" src="images/image21.png" width="30%">
-<img alt="Identity device 2" src="images/image23.png" width="30%">
-<img alt="Identity device 3" src="images/image17.png" width="30%">
+<img alt="Identity device 1" src="/img/zebra-iota-edge-sdk/104/image21.png" width="30%"/>
+<img alt="Identity device 2" src="/img/zebra-iota-edge-sdk/104/image23.png" width="30%"/>
+<img alt="Identity device 3" src="/img/zebra-iota-edge-sdk/104/image17.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md:87]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md [hunk 19]
```diff
@@ -68,50 +68,50 @@ adb shell am startservice -a org.iota.zebra.device_id.intent.action.SCAN -e com.
...
 </p>
 
 After having obtained a new Decentralized Identity, which includes verification methods (i.e. public / private keys), the next step is to get a new Verifiable Credential that will contain all the claims associated with the device (serial number, manufacturer, etc.). Thus, it is first required that the device presents its claims to an issuer. In this case the issuer will be **the organization owning the device**. Such an organization will check for the ownership of the DID and the claims made. And, after a verification process, a new Verifiable Credential will be issued and signed. The issuer of such a Verifiable Credential will be the organization owning the device and the subject the device itself.
 
 In our blueprint application the process described above is conducted as shown below. From the _DeviceID Application_ it is requested a new DeviceID credential. For that purpose, the device generates a QR code that encodes all its claims.
 
 <p align="center">
-<img alt="Request Credential 1" src="images/image16.png" width="30%" align="center">
+<img alt="Request Credential 1" src="/img/zebra-iota-edge-sdk/104/image16.png" width="30%" align="center"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md:95]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md [hunk 20]
```diff
@@ -68,50 +68,50 @@ adb shell am startservice -a org.iota.zebra.device_id.intent.action.SCAN -e com.
...
 </p>
 
-Afterwards, such QR code is scanned by the [*Holder Application*](../102/zebra-iota-edge-sdk-102-tutorial.md) that will verify the claims, the ownership of the DID and will finally issue a new credential.
+Afterwards, such QR code is scanned by the [*Holder Application*](../zebra-iota-edge-sdk/zebra-iota-edge-sdk-102-tutorial.md) that will verify the claims, the ownership of the DID and will finally issue a new credential.
 
 <p align="center">
-<img alt="Device ID Credential" src="images/image8.png" width="30%">
-<img alt="Device ID Claims" src="images/image20.png" width="30%">
+<img alt="Device ID Credential" src="/img/zebra-iota-edge-sdk/104/image8.png" width="30%"/>
+<img alt="Device ID Claims" src="/img/zebra-iota-edge-sdk/104/image20.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md:102]: Same comment

File: docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md [hunk 21]
```diff
@@ -68,50 +68,50 @@ adb shell am startservice -a org.iota.zebra.device_id.intent.action.SCAN -e com.
...
 
 Once the new Credential is ready it is shared from the _Holder Application_ through a scannable DataMatrix code and imported by the _DeviceID application_.
 
 <p align="center">
-<img alt="Onboard device 1" src="images/image4.png" width="30%">
-<img alt="Onboard device 2" src="images/image12.png" width="30%">
-<img alt="Onboard device 3" src="images/image17.png" width="30%">
+<img alt="Onboard device 1" src="/img/zebra-iota-edge-sdk/104/image4.png" width="30%"/>
+<img alt="Onboard device 2" src="/img/zebra-iota-edge-sdk/104/image12.png" width="30%"/>
+<img alt="Onboard device 3" src="/img/zebra-iota-edge-sdk/104/image17.png" width="30%"/>
```
Comment (7opf) [inline docs/tutorials/zebra-iota-edge-sdk/zebra-iota-edge-sdk-104-tutorial.md:110]: Same comment

**Review by 7opf — CHANGES_REQUESTED** [review]
Please address comments, otherwise good to go :)
Thank you

**Review by 7opf — APPROVED** [review]

**Comment by 7opf** [comment]
Review comments will be addressed in a follow up PR

**Pull request merged**